src/Runner.Worker/ActionCommandManager.cs

```diff
@@ -270,12 +270,9 @@ public void ProcessCommand(IExecutionContext context, string line, ActionCommand
                 if (string.Equals(blocked, envName, StringComparison.OrdinalIgnoreCase))
                 {
                     // Log Telemetry and let user know they shouldn't do this
-                    var issue = new Issue()
-                    {
-                        Type = IssueType.Error,
-                        Message = $"Can't update {blocked} environment variable using ::set-env:: command."
-                    };
-                    issue.Data[Constants.Runner.InternalTelemetryIssueDataKey] = $"{Constants.Runner.UnsupportedCommand}_{envName}";
+                    var message = $"Can't update {blocked} environment variable using ::set-env:: command.";
+                    var metadata = new IssueMetadata(Constants.Runner.InternalTelemetryIssueDataKey, $"{Constants.Runner.UnsupportedCommand}_{envName}");
```

Comment:
I defined `IssueMetadata` a data bucket to pass to `CreateIssue` It holds the seldom-used fields of an `Issue`.  It was easier than polluting the `CreateIssue` function signature with those seldom-used fields.